src/scaffoldmaker/meshtypes/meshtype_3d_stellate1.py

```diff
@@ -108,27 +132,88 @@ def generateBaseMesh(cls, region, options):
 
+        markerGroup = findOrCreateFieldGroup(fm, "marker")
+        markerName = findOrCreateFieldStoredString(fm, name="marker_name")
+        markerLocation = findOrCreateFieldStoredMeshLocation(fm, mesh, name="marker_location")
+
+        markerPoints = findOrCreateFieldNodeGroup(markerGroup, nodes).getNodesetGroup()
+        markerTemplateInternal = nodes.createNodetemplate()
+        markerTemplateInternal.defineField(markerName)
+        markerTemplateInternal.defineField(markerLocation)
+
+        # markers with element number and xi position
+        allMarkers = {}
+        if isMouse:
+            xProportion = {}
+            xProportion['ICN'] = 0.9
+            xProportion['VA'] = 0.9
+            xProportion['DA'] = 0.9
+            xProportion['C8'] = 0.9
+            xProportion['T1'] = 0.25
+            xProportion['T2'] = 0.5
+            xProportion['T3'] = 0.75
+            xProportion['TST'] = 1
+            armNumber = {}
+            armNumber['ICN'] = 2
+            armNumber['VA'] = 2
+            armNumber['DA'] = 3
+            armNumber['C8'] = 3
+            armNumber['T1'] = 1
+            armNumber['T2'] = 1
+            armNumber['T3'] = 1
+            armNumber['TST'] = 1
+            nerveAbbrev = list(xProportion.keys())
+            elementIndex = {}
+            xi1 = {}
+            for nerve in nerveAbbrev:
+                elementIndex[nerve] = int(xProportion[nerve] * elementsCount1[armNumber[nerve]-1])
+                xi1[nerve] = 1 if xProportion[nerve] == 1 else xProportion[nerve] * elementsCount1[armNumber[nerve]-1] - elementIndex[nerve]
+                elementIndex[nerve] += 1 if xProportion[nerve] < 1 else 0
+                j = 10
+
+            allMarkers = { "Inferior cardiac nerve" : {"elementID": elementIndex['ICN']+2*elementsCount1[0], "xi": [xi1['ICN'], 0.0, 0.5]},
+                            "Ventral ansa subclavia" : {"elementID": elementIndex['VA']+2*elementsCount1[0]+elementsCount1[1], "xi": [xi1['VA'], 1.0, 0.5]},
+                            "Dorsal ansa subclavia" : {"elementID": elementIndex['DA']+2*(elementsCount1[0]+elementsCount1[1]), "xi": [xi1['DA'], 0.0, 0.5]},
+                            "Cervical spinal nerve 8" : {"elementID": elementIndex['C8']+2*(elementsCount1[0]+elementsCount1[1])+elementsCount1[2], "xi": [xi1['C8'], 1.0, 0.5]},
+                            "Thoracic spinal nerve 1" : {"elementID": elementIndex['T1'], "xi": [xi1['T1'], 0.0, 0.5]},
+                            "Thoracic spinal nerve 2" : {"elementID": elementIndex['T2'], "xi": [xi1['T2'], 0.0, 0.5]},
+                            "Thoracic spinal nerve 3" : {"elementID": elementIndex['T3'], "xi": [xi1['T3'], 0.0, 0.5]},
+                            "Thoracic sympathetic nerve trunk" : {"elementID": elementIndex['TST'], "xi": [xi1['TST'], 1.0, 0.5]},
+                           }
+
+        # arm group annotations for user
+        armNames, faceNames = getUnlabelledStellateArmNames(armCount)
```

Comment:
Use 'Term' instead of Name for all these as they are tuple (name, id). I expect 'Name' to only be a simple string.
Also, if you're not using one of the return values it's better to follow one of these idioms to ignore it:
```
armNames, _ = getUnlabelledStellateArmNames(armCount)
armNames = getUnlabelledStellateArmNames(armCount)[0]
```
The last one is preferred as it will work even if you add more return values.